objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
What you say here contradicts what you just said one comment above. If CSV on a one-dimensional list should be the same as plain text, then it should obviously be comma-separated. Otherwise, it needs two different names.

We will never manage to please everyone though, because separators are absolutely dependent on each case. I explicitly didn't add `plain text` at first to avoid discussions about separators. You prefer newlines, Joan may prefer semicolons, I'll vote for commas, and this will never be pulled.

This is getting way too long now. I contributed this because I had already written it for myself and it costed ~0 to issue a pull request, but I don't want to spend another morning tweaking it.

Feel free to change it though! :)